assets/styles/vars.pcss

```diff
@@ -1,5 +1,28 @@
 :root {
-  --layout-content-max-width: 1024px;
+  --layout-container-max-width: 1180px;
+  --layout-padding-horizontal: 20px;
+  --layout-header-height: 60px;
+
+  --color-background-hover: #F0F5F9;
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
?

Reply:
above: refactored, below: to be refactored

Reply:
🫤

Reply:
I've removed empty lines